.azure/templates/build-config-user.yml

```diff
@@ -32,6 +32,12 @@ jobs:
       version: 3.1.302
       installationPath: $(Agent.ToolsDirectory)/dotnet
 
+  - pwsh: |
+      Choco-Install -PackageName strawberryperl
+      Write-Host "##vso[task.prependpath]C:\Strawberry\perl\bin"
+    displayName: Install Perl
+    condition: and(eq('${{ parameters.image }}', 'windows-2022'), eq('${{ parameters.tls }}', 'openssl'))
```

Comment:
Can we scope this to only when necessary?

Reply:
It is. Thats what the condition does.

Reply:
Ah, duh. I was looking for it at the top for some reason.